types/global.d.ts

```
@@ -0,0 +1,12 @@
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This export makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)